metaflow/plugins/pypi/conda_environment.py

```diff
@@ -65,7 +65,7 @@ def validate_environment(self, echo, datastore_type):
         micromamba = Micromamba()
         self.solvers = {"conda": micromamba, "pypi": Pip(micromamba)}
 
-    def init_environment(self, echo):
+    def init_environment(self, echo, only_steps=[]):
```

Comment:
@romain-intel I'd like your thoughts on this mechanism.

The reason we need something akin to this is to support the environment fallback logic in `docker_environment.py`.

1. docker env gathers a list of steps that it can not act on (conda/pypi but not executing remotely)
2. for the incompatible steps we want to fallback to the conda implementation, and initialize accordingly
3. the only_steps is meant to skip those steps that we were able to bake a docker image for

Would this be a good addition to the environment init flow. Any better alternatives to this?